templates/cfp/proposals.html

```diff
@@ -12,8 +12,13 @@ <h2>Your proposals</h2>
   {% for proposal in proposals %}
   <tr class="{{- loop.cycle('odd', 'even') }}">
     <td>
-      <h4><a href="{{ url_for('.edit_proposal', proposal_id=proposal.id) }}">{{ proposal.title }}</a></h4>
-      <p>{{ proposal.human_type | title }}</p>
+      <h4><a href="{{ url_for('.edit_proposal', proposal_id=proposal.id) }}">{{ proposal.title }}{% if proposal.state in ('edit', 'new') %} [edit]{% endif %}</a></h4>
```

Comment:
This [edit] flag only makes sense to us. It's probably clearer to disable the fieldsets and Update button in `edit-form.html` if that's the intention (I'm guessing people should still be able to message about a withdrawn proposal).

Reply:
The intention on this was just to make it clear that they could click through on the title to edit their submission (because I don't think it's super obviously a URL)